index.css

```diff
@@ -10,6 +10,33 @@ body {
 /*================================================1 - BANNER ========================================*/
 /*================================================================================================*/
 
+/*****************************************   Website header  **************************************/
+.banner-section .web-header {
+  width: 100%;
+  background: black;
+  position: absolute; /* we need it, because otherwise it takes the place below image */
+  top: 0px;
+  backdrop-filter: blur(2px);
+}
+.banner-section .web-header > * {
+  display: inline-block;
+  margin: 14px 0px 14px 220px;
```

Comment:
please apply it only for div, not for all elements
Why? because in the next lines use set margin: 0